packages/legacy/core/__tests__/screens/__snapshots__/Home.test.tsx.snap

```diff
@@ -115,7 +117,80 @@ exports[`displays a home screen renders correctly 1`] = `
     </View>
     <View
       onLayout={[Function]}
-    />
+    >
+      <View>
+        <View
+          style={
+            Object {
+              "paddingBottom": 75,
+              "paddingHorizontal": 25,
+            }
+          }
+        >
+          <View
+            style={
+              Array [
+                Object {
+                  "alignItems": "center",
+                  "justifyContent": "center",
+                  "marginHorizontal": 25,
+                  "marginTop": 35,
+                },
+              ]
+            }
+          >
+            <Text
+              adjustsFontSizeToFit={true}
+              style={
+                Array [
+                  Object {
+                    "color": "#FFFFFF",
+                    "fontSize": 38,
+                    "fontWeight": "bold",
+                  },
+                  Object {
+                    "marginBottom": 20,
+                    "marginTop": 25,
+                  },
+                ]
+              }
+            >
+              Home.Welcome
+            </Text>
+          </View>
+          <View
+            style={
+              Array [
+                Object {
+                  "alignItems": "center",
+                  "justifyContent": "center",
+                  "marginHorizontal": 25,
+                  "marginTop": 35,
+                },
+              ]
+            }
+          >
+            <Text
+              style={
+                Array [
+                  Object {
+                    "color": "#FFFFFF",
+                    "fontSize": 18,
+                    "fontWeight": "normal",
+                  },
+                  Object {
+                    "marginTop": 25,
+                    "textAlign": "center",
+                  },
+                ]
+              }
+            >
+              Home.NoCredentials
```

Comment:
Why did this snapshot change? Never seen "No credentials" on the home screen

Reply:
+1

Reply:
The `you have no credentials in your wallet` is supposed to be there. IDk why the previous configuration didn't have it. maybe some old default configuration was blocking it? 
<img width="393" alt="image" src="https://github.com/user-attachments/assets/4721fe9d-dca2-40f7-bf77-1664136e07f5">

Reply:
I think the BC Wallet config overrides it so maybe the old bifold config we had for the tests did the same thing